src/ex4/tasks/task.txt

```diff
@@ -0,0 +1,4 @@
+asdf
```

Comment:
not relevant file for the PR

Reply:
Do i have options to not include file in PR or i need to add this file to .gitignore?